packages/web-components/fast-components/src/listbox/index.ts

```diff
@@ -1,6 +1,25 @@
-import { Listbox, listboxTemplate as template } from "@microsoft/fast-foundation";
+import {
+    Listbox as FoundationListbox,
+    listboxTemplate as template,
+} from "@microsoft/fast-foundation";
+import { fillColor, neutralLayerFloating } from "../design-tokens";
 import { listboxStyles as styles } from "./listbox.styles";
 
+/**
+ * The FAST listbox class
+ * @public
+ */
+export class Listbox extends FoundationListbox {
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        fillColor.setValueFor(this, neutralLayerFloating);
```

Comment:
We don't need this in the listbox, only the select and combobox where that's a flyout.

Reply:
removed